Mergin/project_settings_widget.py

```diff
@@ -85,13 +85,16 @@ def __init__(self, parent=None):
         idx = self.cmb_tracking_precision.findData(mode) if ok else 1
         self.cmb_tracking_precision.setCurrentIndex(idx)
 
-        enabled, ok = QgsProject.instance().readBoolEntry("Mergin", "MapAnnotations/Enabled")
+        enabled, ok = QgsProject.instance().readBoolEntry("Mergin", "MapSketches/Enabled")
```

Comment:
Sorry, my bad, these entries should bear the name of the feature -> `MapSketching/*`